src/screens/Transactions/Transactions.jsx

```diff
@@ -17,24 +16,18 @@ const {
 } = C;
 
 const Transactions = ({ route: { params: { account: { hash } } = {} } = {}, navigation = {} }) => {
-  const scrollview = useRef(null);
   const { accounts = [], settings: { baseCurrency } = {} } = useStore();
 
-  const { height } = useWindowDimensions();
-
   const [dataSource, setDataSource] = useState({});
-  const [scrollQuery, setScrollQuery] = useState(false);
   const [txs, setTxs] = useState([]);
+  const [page, setPage] = useState(2);
```

Comment:
why `2`?

Reply:
To have the first 40 items rendered at the first time